src/averageAges.js

```diff
@@ -14,12 +14,19 @@
  *
  * @return {number}
  */
+function calculateAverage(array) {
+  return array.reduce((sum, diff) => sum + diff, 0) / array.length;
+}
+
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  let men = people.filter(person => person.sex === 'm');
+
+  century && (men = men.filter(person =>
+    Math.ceil(person.died / 100) === century));
```

Comment:
don't you think it looks more readable?;)

```suggestion
  if (century) {
    men = men.filter(person =>
      Math.ceil(person.died / 100) === century));
  }
```